Samples/YOLOv4ObjectDetection/README.md

```diff
@@ -0,0 +1,27 @@
+# YOLOv4 Object Detection Sample
+
+This UWP C# app uses the YOLOv4 model to detect objects in video streams. 
+
+The [YOLOv4 model](https://github.com/hunglc007/tensorflow-yolov4-tflite) is trained with [Tensorflow-DML](https://github.com/microsoft/tensorflow-directml), and converted to ONNX through [tf2onnx](https://github.com/onnx/tensorflow-onnx)

```

Comment:
do we need to add any licensing text due to including the yolo model?

Reply:
I think it should be fine. That Github Repo use MIT license. And our Repo is stated to use the same license.

---
In reply to: [568059295](https://github.com/microsoft/Windows-Machine-Learning/pull/376#discussion_r568059295) [](ancestors = 568059295)